server/utils/applicationUtils.ts

```diff
@@ -86,4 +86,42 @@ const getPage = (taskName: string, pageName: string) => {
   return Page
 }
 
-export { getTaskStatus, taskLink, getCompleteSectionCount, getResponses, getResponseForPage, getPage }
+const getArrivalDate = (application: Application): string | null => {
```

Comment:
I do wonder if this is something we could/should do in the API at the point of submission, but this is OK for now

Reply:
I've added a card https://trello.com/c/U6OqS6Zb/928-get-application-arrival-date-at-point-of-submission

Reply:
Yeah 100% agree, my understanding was this was a workaround for the time being